rebar.config

```diff
@@ -1,37 +1,11 @@
-%% Known library limitations that make support for Erlang/OTP
-%% versions R15 or earlier difficult:
-%%
-%% uuid: Dep on quickrand
-%% quickrand: Bogus (IMHO) inability to adapt to R15 environment.
 {require_otp_vsn, "R16|17"}.
 
 {deps,
  [
-  %% NOTE: some basho_bench drivers make use of pre-folsom stats
-  %%       that permitted float type counter increments.  Thus
-  %%       we currently rely on a custom branch of folsom that
-  %%       has a useful-but-icky hack that allows fractional
-  %%       increments.  If someone wants to take advantage of a
-  %%       new folsom feature, that desire + float incr must be
-  %%       weighed.
-  {node_package, "2.0.*", {git, "git://github.com/basho/node_package", {tag, "2.0.0"}}},
-  {folsom, ".*", {git, "git://github.com/cloudant/couchdb-folsom.git", {branch, "45918-pluggable-storage-engines"}}},
-  {lager, "2.*", {git, "git://github.com/basho/lager", {tag, "2.1.0"}}},
-  {ibrowse, ".*",
-   {git, "git://github.com/cmullaparthi/ibrowse.git", {tag, "v4.0.2"}}},
```

Comment:
The perf HTTP benchmarks require ibrowse - until we have cloudant_bench better defined it may make sense to leave that in basho_bench.

Glad to see mochiweb going - that was drawing in a lot of other stuff for the sake of a couple of encode/decode functions used in drivers you've already removed